tests/test_marathon_serviceinit.py

```diff
@@ -429,26 +429,74 @@ def test_status_smartstack_backends_normal():
 def test_status_smartstack_backends_different_nerve_ns():
     service = 'my_service'
     instance = 'my_instance'
-    cluster = 'fake_cluster'
     different_ns = 'other_instance'
-    normal_count = 10
-    tasks = mock.Mock()
-    with mock.patch('paasta_tools.marathon_tools.read_namespace_for_service_instance', autospec=True) as read_ns_mock:
-        read_ns_mock.return_value = different_ns
+    service_instance = compose_job_id(service, different_ns)
```

Comment:
I know this is silly, but can you drop this line and make the assertion be:
```
service_instance="my_service.other_namespace"
```

And just hard code it in the test.

The reasoning here is that I want to make it very obvious that the function is called with the other namespace, and not the normal "service_instance"